### PR TITLE
[MM-52745] server/Makefile: update morph usage in new-migration target

### PR DESCRIPTION
**Pull request by isacikgoz**
#### Summary
Use `new script` sub command instead of `generate` for the `new-migration` target.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-52745

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by mgdelacroix — APPROVED** [review]
LGTM! :+1:

File: server/Makefile [hunk 1]
```diff
@@ -338,12 +338,12 @@ generate-worktemplates: ## Generate work templates
 	$(GO) generate $(GOFLAGS) ./channels/app/worktemplates
 
 new-migration: ## Creates a new migration. Run with make new-migration name=<>
-	$(GO) install github.com/mattermost/morph/cmd/morph@master
+	$(GO) install github.com/mattermost/morph/cmd/morph@1e0640c
```
[the remark this answers is not in the document]
Comment (isacikgoz) [reply server/Makefile:341]: Yeah, but it's actually not good to use `master` I think due to unintended changes over time.

[the remark this answers is not in the document]
Comment (agnivade) [reply server/Makefile:341]: Ok, then perhaps we could tag a release. But non-blocking.

**Pull request merged**